gobblin-service/src/main/java/org/apache/gobblin/service/modules/restli/GobblinServiceFlowExecutionResourceHandler.java

```diff
@@ -71,15 +71,14 @@ public FlowExecution get(ComplexResourceKey<FlowStatusId, EmptyRecord> key) {
   }
 
   @Override
-  public UpdateResponse resume(ComplexResourceKey<FlowStatusId, EmptyRecord> key) {
+  public void resume(ComplexResourceKey<FlowStatusId, EmptyRecord> key) {
```

Comment:
What's the reason to use ComplexResourceKey<FlowStatusId, EmptyRecord> instead of FlowStatusId directly?

Reply:
Not really a reason in particular, restli just gives us the ID as a `ComplexResourceKey` and at some point we have to call getKey to get the `FlowStatusId ` from it.

Reply:
ok. I see that it's the same in other methods here. Let's keep it consistent then.